src/mlbstandings/service.py

```
@@ -0,0 +1,37 @@
+import os
+from .app import app
+
+from gunicorn.app.base import BaseApplication
+
+class StandaloneApplication(BaseApplication):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)